guilford_court_commitment.yaml

```diff
@@ -0,0 +1,408 @@
+        type: "ol.recognizedstartdate"
+        column: "P&P CASE INTAKE DATE"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss.S"]
+          timezone: "America/New_York"
+      ol.status: # p&p commitment status flag
+        type: "ol.status"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "P&P COMMITMENT STATUS FLAG"
+          pattern: "\\?+"
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "P&P COMMITMENT STATUS FLAG"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: ""
+      general.datetime: # p&p commitment status date
+        type: "general.datetime"
+        column: "P&P COMMITMENT STATUS DATE"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss.S"]
+          timezone: "America/New_York"
+      ol.durationdays: # total length of supervision
+        type: "ol.durationdays"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "TOTAL LENGTH OF SUPERVISION"
+          pattern: "\\?+"
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "TOTAL LENGTH OF SUPERVISION"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: ""
+      ol.type:
+        type: "ol.type"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "TYPE OF OLD PE DATE CODE"
+          pattern: "\\?+"
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "TYPE OF OLD PE DATE CODE"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: ""
+      ol.newperiod:
+        type: "ol.newperiod"
+        column: "NEW PERIOD OF SUPERVISION FLAG"
+    name: "supervisions"
+
+  locations:
+    fqn: "ol.location"
+    entitySetName: "GuilfordLocations"
+    propertyDefinitions:
+      general.id: # most serious offense code
+        type: "general.id"
+        column: "CO OF CONV MOST SERIOUS OFFNSE"
+      ol.county: # county of most serious offense
+        type: "ol.county"
+        column: "CO OF CONV MOST SERIOUS OFFNSE"
+    conditions:
+    - !<conditions.BooleanRegexCondition>
+      column: "CO OF CONV MOST SERIOUS OFFNSE"
+      pattern: "\\?+"
+      reverse: true
+    name: "locations"
+
+
+
+associationDefinitions:
+ 
+
+
+  subjectof:
+    fqn: "ol.subjectof"
+    entitySetName: "GuilfordSubjectOf"
+    src: "offenders"
+    dst: "jailstays"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.HashTransform>
+          columns: ["COMMITTED FIRST NAME", "COMMITTED MIDDLE NAME", "COMMITTED LAST NAME", "COMMITTED NAME SUFFIX", "OFFENDER NC DOC ID NUMBER", "OFFENDER ADMISSION/INTAKE DATE"]
+          hashFunction: "sha256"
+    name: "subjectof"
+
+  sentencedwith:
```

Comment:
Also it would be ideal if the associations here merged with those here, in the case of the same person and sentence - therefore the associations should be constructed identically. Please check if there are any others in that older flight that you should copy in terms of structure. https://github.com/Lattice-Works/Guilford-Co-NC/blob/master/guilford_inmates_dps.yaml#L965